components/Grid/Grid.css

```diff
@@ -0,0 +1,63 @@
+:root{
+    --pink:rgb(255, 192, 229);
+    --dk-pink:rgb(251, 111, 181);
+    --red:rgb(255, 137, 137);
+    --dk-red:rgb(206, 21, 21);
+    --blue:rgb(176, 229, 255);
+    --dk-blue:rgb(104, 197, 255);
+    --purple:rgb(223, 169, 223);
+    --dk-purple:rgb(202, 64, 202);
+
+}
```

Comment:
Just a suggestion, you can create a common variable stylesheet for all your variables.